spec/windows_spec.rb

```diff
@@ -37,18 +37,20 @@
     expected_windows = [
       RAutomation::Window.new(:pid => @pid1),
     ]
-    should have_all_windows(expected_windows, windows)
+    expect(has_all_windows?(expected_windows, windows)).to be true
   end
 
   it "Windows#windows with parameters returns all matching windows" do
-    windows = RAutomation::Window.new(:title => SpecHelper::DATA[:window1_title]).windows(:title => SpecHelper::DATA[:window2_title])
-    windows.should be_a(RAutomation::Windows)
+    windows = RAutomation::Window.new(:title => SpecHelper::DATA[:window1_title])
+                                 .windows(:title => SpecHelper::DATA[:window2_title])
+    expect(windows).to be_a(RAutomation::Windows)
 
-    windows.size.should == 1
+    expect(windows.size).to eq(2) # this always returns 2?
```

Comment:
Are you saying that it returns 2 even without any of the changes in this branch (e.g. on the master branch code) for you?

Reply:
Sorry the comment is incorrect. This windows.size call always returns 1. And yes, also on the current master branch.